### ISSUE TITLE
It doesn't work on windows

### ISSUE THREAD

**Issue by andyhu** [nuxt/nuxt.js#9]
Here below are the log messages:
```
ERROR in ./~/babel-loader/lib?presets[]=es2015&presets[]=stage-2!./~/vue-loader/lib/selector.js?type=script&index=0!./.nuxt/App.vue
Module not found: Error: Can't resolve 'I:Projectscms
ode_modules
uxtlibappcomponents
uxt-error-debug.vue' in 'I:\Projects\cms\.nuxt'
 @ ./~/babel-loader/lib?presets[]=es2015&presets[]=stage-2!./~/vue-loader/lib/selector.js?type=script&index=0!./.nuxt/App.vue 7:62-141
 @ ./.nuxt/App.vue
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi app
```
The main problem is that some `import`s / `System.import`s in generated code under `.nuxt` folder doesn't escape the directory path `\`:
```js
const _eb75b49c = process.BROWSER ? () => System.import('I:\Projects\test\pages\index.vue') : require('I:\Projects\test\pages\index.vue')
```

```js
import NuxtError from 'I:\Projects\test\node_modules\nuxt\lib\app\components\nuxt-error-debug.vue'
```

Can we use relative path and forward slash `/` or escape backslash in the path?

**Comment by Atinux** [comment]
Hi @andyhu 

It's now working in version [v0.5.1](https://github.com/nuxt/nuxt.js/releases/tag/v0.5.1)

**Comment by andyhu** [comment]
Thanks @Atinux for the prompt response as well as your awesome job. Much appreciated!

**Comment by andyhu** [comment]
Still have 1 issue:
```js
require('~plugins\\\\element-ui.js')
```
now plugins are double escaped

**Comment by Atinux** [comment]
I will take a look, sorry about this.

**Comment by Atinux** [comment]
Fixed in @0.6.1
